tools/rosgraph/src/rosgraph/rosenv.py

```diff
@@ -69,7 +69,10 @@ def get_master_uri(env=None, argv=None):
                 if not val:
                     raise ValueError("__master remapping argument '%s' improperly specified"%arg)
                 return val
-        return env[ROS_MASTER_URI]
+        if ROS_MASTER_URI in env:
```

Comment:
You could use a one-liner instead:
`env.get(ROS_MASTER_URI, 'http://localhost:11311')`